index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Tesla</title>
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="/style.css">
+</head>
+	<body>
+		<header>
+			<div class="menu">
+				<nav class="menu__bar">
+					<img src="" alt="tesla_wordmark_red" class=menu__bar_watermark>
+					<a href="" class="menu__bar_item">Model S</a>
+					<a href="" class="menu__bar_item">Model X</a>
+					<a href="" class="menu__bar_item">Model 3</a>
+					<a href="" class="menu__bar_item">Roadster</a>
+					<a href="" class="menu__bar_item">Energy</a>
+					<a href="" class="menu__bar_item">Магазин</a>
+					<a href="" class="menu__bar_item">Войти</a>
+					<img src="" alt="vector" class="menu__bar_vector">
+				</nav>
+			</div>
+		</header>
+		<main>
+			<div class="main-header">
+				<h1 class="main-header__title">Model S</h1>
+				<h4 class="main-header__subtitle">Добро пожаловать в будущее!</h4>
+				<img src="" alt="Tesla-Wordmark-Red" class="main-wrapper__background-watermark">
+				<img src="" alt="Tesla-car" class="main-wrapper__image">
+			</div>
+			<div class="main-bottom-wrapper">
+				<div class="main-bottom-wrapper__block1">
```

Comment:
I'd suggest to use `id="block1"`, `id="block2` etc here and keep the class generic as `class="main-bottom-wrapper__block"`; I think the classes should be used for something general, and everything specific should be addressed using ids (even if it goes against classic BEM in a way)